File: app/src/main/java/com/sd/dialogview/handler/AppDialogMenuViewHandler.java
package com.sd.dialogview.handler;

import com.sd.lib.dialogview.DialogMenuView;
import com.sd.lib.dialogview.core.handler.IDialogMenuViewHandler;

public class AppDialogMenuViewHandler implements IDialogMenuViewHandler
{
    @Override
    public int getContentViewResId(DialogMenuView dialogView)
    {
        return 0;
    }

    @Override
    public void onContentViewChanged(DialogMenuView dialogView)
    {

    }

    @Override
    public void onAttachedToWindow(DialogMenuView dialogView)
    {
        dialogView.setTextTitle("hook title");
    }

    @Override
    public void onDetachedFromWindow(DialogMenuView dialogView)
    {

    }
}

File: app/src/main/java/com/sd/demo/dialogview/handler/AppDialogMenuViewHandler.kt
package com.sd.demo.dialogview.handler

import com.sd.lib.dialogview.DialogMenuView
import com.sd.lib.dialogview.core.handler.IDialogMenuViewHandler

class AppDialogMenuViewHandler : IDialogMenuViewHandler {
    override fun getContentViewResId(dialogView: DialogMenuView): Int {
        return 0
    }

    override fun onContentViewChanged(dialogView: DialogMenuView) {

    }

    override fun onAttachedToWindow(dialogView: DialogMenuView) {
        dialogView.setTextTitle("hook title")
    }

    override fun onDetachedFromWindow(dialogView: DialogMenuView) {

    }
}